packages/apps-config/src/api/spec/uart.ts

```diff
@@ -159,8 +201,9 @@ const definitions: OverrideBundleDefinition = {
         },
         ReFungibleItemType: {
           Collection: 'u64',
-          Owner: 'Vec<Ownership>',
-          Data: 'Vec<u8>'
+          Owner: 'Vec<Ownership<AccountId>>',
```

Comment:
```suggestion
          Owner: 'Vec<Ownership>',
```

Unlike on the Rust side, the types parser is not clever enough to be passed a generic parameter.